tests/ContextIntegrationTests.js

```diff
@@ -0,0 +1,318 @@
+/**
+ * GPII Context Integration Tests
+ *
+ * Copyright 2014 Lucendo Development Ltd.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/kettle/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    kettle = fluid.registerNamespace("kettle"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.registerNamespace("gpii.test.contextIntegration");
```

Comment:
Well ok we need to review this globally since we are ending up with quite a mixture of levels now. All the contents of "testing" seem to be in gpii.test which we may as well keep - but increasingly they are filling up with actual test cases